.gitattributes

```diff
@@ -0,0 +1 @@
+text eol=auto
```

Comment:
@magnonasc , double-check the format here. I'm pretty sure it should be `* text=auto`. Look at the Github documentation, the `.gitattributes` documentation, and compare with the `GlobalMentor` `.gitattributes` file.